content/zh-cn/docs/concepts/services-networking/service-traffic-policy.md

```diff
@@ -1,16 +1,23 @@
 ---
 title: 服务内部流量策略
 content_type: concept
-weight: 45
----
-<!-- 
+weight: 75
+description: >-
+  如果集群中的两个 Pod 想要通信，并且两个 Pod 实际上都在同一个节点上运行，
+  则**服务内部流量策略（Service Internal Traffic Policy）**将网络流量保持在该节点内。 
```

Comment:
[The preview](https://deploy-preview-37009--kubernetes-io-main-staging.netlify.app/zh-cn/docs/concepts/services-networking/) of `**bold**` doesn't seems right.